src/Symfony/Bundle/FrameworkBundle/Command/RouterDebugCommand.php

```diff
@@ -125,4 +132,28 @@ private function findRouteNameContaining(string $name, RouteCollection $routes):
 
         return $foundRoutesNames;
     }
+
+    private function sortRoutes(RouteCollection $routes, string $propertyName): RouteCollection
+    {
+        $validOptions = ['', 'priority', 'name', 'path'];
```

Comment:
not needed

Reply:
Removed and changed last `elseif` to take care of the `'priority'` case.